build.xml

```diff
@@ -57,6 +57,11 @@
         <copy todir="${build.project.dir}">
             <fileset dir="${src.dir}" excludes="**/*.java"/>
         </copy>
+
+        <!-- Distribute socket with LICENSE only -->
```

Comment:
This comment is left over from the [`qz-print`](https://github.com/qzind/qz-print/blob/1.9/build.xml#L248-L256) split applet/socket builds and can now be removed.